app/src/org/commcare/views/media/AudioPlaybackButton.java

```diff
@@ -260,4 +294,36 @@ private void pausePlaying() {
         currentState = MediaState.Paused;
         refreshAppearance();
     }
+
+    private void animateProgress(int milliPosition, int milliDuration) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
+            ProgressBar progressBar = (ProgressBar)findViewById(R.id.circular_progress_bar);
+            int startPosition = 0;
+            int progressDuration = 500;
```

Comment:
Hm, I'm still confused. Does the call to `animation.setDuration(milliDuration)` not just override the `500` value that was set in the previous line? Or do those "durations" actually mean different things?